Mathlib/Algebra/Order/Ring/Finset.lean

```diff
@@ -1,34 +1,48 @@
 /-
-Copyright (c) 2022 Eric Wieser. All rights reserved.
+Copyright (c) 2022 Eric Wieser, Yaël Dillies, Andrew Yang. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Eric Wieser
+Authors: Eric Wieser, Yaël Dillies, Andrew Yang
 -/
-import Mathlib.Algebra.Order.Ring.Defs
+import Mathlib.Algebra.Order.Field.Canonical.Defs
+import Mathlib.Algebra.Order.Floor
 import Mathlib.Data.Finset.Lattice.Fold
 
 /-!
-# Algebraic properties of finitary supremum
+# `Finset.sup` in a ring
```

Comment:
Now this file is just about casts interacting with `Finset.sup`, so maybe a docstring describing it that way would be nice